schema-lending.graphql

```diff
@@ -80,6 +83,12 @@ enum LendingType {
 
   " Pooled protocols pool all users assets into a single market "
   POOLED
+
+  " Protocols that only allow KYC'd users "
+  WHITELIST_ONLY
```

Comment:
I would separate this out into another field (e.g. `PermissionType`) as it's a different dimension

Reply:
created